### PR TITLE
Fix handling of RV_LOCK and RV_SNAP for windows

### PR DESCRIPTION
**Pull request by rverst**
- keys were not unregistered properly

## Types of Changes

- [ ] Core
- [x] Bugfix
- [ ] New feature
- [ ] Enhancement/optimization
- [ ] Keyboard (addition or update)
- [x] Keymap/layout/userspace (addition or update)
- [ ] Documentation

## Checklist

- [x] My code follows the code style of this project: [**C**](https://docs.qmk.fm/#/coding_conventions_c), [**Python**](https://docs.qmk.fm/#/coding_conventions_python)
- [x] I have read the [**PR Checklist** document](https://docs.qmk.fm/#/pr_checklist) and have made the appropriate changes.
- [ ] My change requires a change to the documentation.
- [ ] I have updated the documentation accordingly.
- [x] I have read the [**CONTRIBUTING** document](https://docs.qmk.fm/#/contributing).
- [ ] I have added tests to cover my changes.
- [x] I have tested the changes and verified that they work and don't break anything (as well as I can manage).

### CODE REVIEW

File: users/rverst/rverst.c [hunk 1]
```diff
@@ -205,7 +205,7 @@ bool process_record_user(uint16_t keycode, keyrecord_t *record) {
             } else if (mode == WINDOWS || mode == WINDOWS_UNI) {
                 register_code(KC_LGUI);
                 tap_code(KC_L);
-                register_code(KC_LGUI);
+                unregister_code(KC_LGUI);
```
Comment (fauxpark) [inline users/rverst/rverst.c:208]: This can just be:
```c
tap_code16(G(KC_L));
```

File: users/rverst/rverst.c [hunk 2]
```diff
@@ -230,8 +230,8 @@ bool process_record_user(uint16_t keycode, keyrecord_t *record) {
                 register_code(KC_LGUI);
                 register_code(KC_LSFT);
                 tap_code(KC_S);
-                register_code(KC_LSFT);
-                register_code(KC_LGUI);
+                unregister_code(KC_LSFT);
+                unregister_code(KC_LGUI);
```
Comment (fauxpark) [inline users/rverst/rverst.c:234]: And again:
```c
tap_code16(S(G(KC_S)));
```
etc.

**Review by fauxpark — CHANGES_REQUESTED** [review]

File: users/rverst/rverst.c (hunk 2 above)
Comment (rverst) [reply users/rverst/rverst.c:234]: Oh, I must have missed that in the docs. I'll change that later, thanks.

File: users/rverst/rverst.c [hunk 3]
```diff
@@ -197,15 +197,9 @@ bool process_record_user(uint16_t keycode, keyrecord_t *record) {
         // Lock computer
         case RV_LOCK:
             if (mode == MAC || mode == MAC_UNI) {
-                register_code(KC_LGUI);
-                register_code(KC_LCTL);
-                tap_code(KC_Q);
-                unregister_code(KC_LCTL);
-                unregister_code(KC_LGUI);
+                tap_code16(G(S(KC_Q)));
```
Comment (fauxpark) [inline users/rverst/rverst.c]:
```suggestion
                tap_code16(C(G(KC_Q)));
```

File: users/rverst/rverst.c [hunk 4]
```diff
@@ -215,23 +209,15 @@ bool process_record_user(uint16_t keycode, keyrecord_t *record) {
                 if (ls) unregister_code(KC_LSFT);
                 if (rs) unregister_code(KC_RSFT);
 
-                register_code(KC_LGUI);
-                register_code(KC_LSFT);
                 if (as)
```
Comment (fauxpark) [inline users/rverst/rverst.c:212]:
```suggestion
```

File: users/rverst/rverst.c [hunk 5]
```diff
@@ -215,23 +209,15 @@ bool process_record_user(uint16_t keycode, keyrecord_t *record) {
...
-                    tap_code(KC_5);
+                    tap_code16(G(S(KC_5)));
                 else
```
Comment (fauxpark) [inline users/rverst/rverst.c]:
```suggestion
```

File: users/rverst/rverst.c [hunk 6]
```diff
@@ -215,23 +209,15 @@ bool process_record_user(uint16_t keycode, keyrecord_t *record) {
...
-                    tap_code(KC_4);
-                unregister_code(KC_LSFT);
-                unregister_code(KC_LGUI);
+                    tap_code16(G(S(KC_4)));
```
Comment (fauxpark) [inline users/rverst/rverst.c:213]:
```suggestion
                    tap_code16(G(S(as ? KC_5 : KC_4)));
```

**Review by fauxpark — CHANGES_REQUESTED** [review]

File: users/rverst/rverst.c (hunk 4 above)
Comment (fauxpark) [inline users/rverst/rverst.c:212]:
```suggestion
```

File: users/rverst/rverst.c [hunk 7]
```diff
@@ -215,23 +209,15 @@ bool process_record_user(uint16_t keycode, keyrecord_t *record) {
                 if (ls) unregister_code(KC_LSFT);
                 if (rs) unregister_code(KC_RSFT);
 
-                register_code(KC_LGUI);
-                register_code(KC_LSFT);
                 if (as)
-                    tap_code(KC_5);
+                    tap_code16(G(S(KC_4)));
                 else
```
Comment (fauxpark) [inline users/rverst/rverst.c:214]:
```suggestion
```

File: users/rverst/rverst.c [hunk 8]
```diff
@@ -215,23 +209,15 @@ bool process_record_user(uint16_t keycode, keyrecord_t *record) {
...
-                    tap_code(KC_4);
-                unregister_code(KC_LSFT);
-                unregister_code(KC_LGUI);
+                    tap_code16(G(S(KC_5)));
```
Comment (fauxpark) [inline users/rverst/rverst.c:215]:
```suggestion
                    tap_code16(G(S(as ? KC_4 : KC_5)));
```

**Review by fauxpark — CHANGES_REQUESTED** [review]

**Review by drashna — APPROVED** [review]

**Review by fauxpark — APPROVED** [review]

**Comment by drashna** [comment]
Thanks!

**Pull request merged**